content/academy/web_scraping_for_beginners/challenge/initializing_and_setting_up.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Initializing & setting up
+description: When you collect links from a web page, you often end up with a lot of irrelevant URLs. Learn how to filter the links to only keep the ones you need.
+menuWeight: 1
+paths:
+- web-scraping-for-beginners/challenge/initializing-and-setting-up
+---
+
+# Initialization & setting up
+
+The Crawlee CLI makes it extremely easy for us to set up a project in Crawlee and hit the ground running. Navigate to the directory you'd like your project's folder to live, then open up a terminal instance and run the following command:
+
+```shell
+npx crawlee create demo-actor
+```
+
+> You don't have to call it **demo-actor**, but that's what we'll be calling it in this tutorial.
```

Comment:
I wouldn't call it demo-**actor**. It's the open-source part of the course so actors are irrelevant here.
```suggestion
npx crawlee create amazon-crawler
```

> You don't have to call it **amazon-crawler**, but that's what we'll be calling it in this tutorial.
```